icu4c/source/test/intltest/unitstest.cpp

```diff
@@ -301,10 +340,15 @@ void unitsTestDataLineFn(void *context, char *fields[][2], int32_t fieldCount, U
     StringPiece commentConversionFormula = trimField(fields[3]);
     StringPiece utf8Expected = trimField(fields[4]);
 
-    UNumberFormat *nf = unum_open(UNUM_DEFAULT, NULL, -1, "en_US", NULL, pErrorCode);
+    UNumberFormat *nf = unum_open(UNUM_DEFAULT, NULL, -1, "en_US", NULL, status);
+    if (status.errIfFailureAndReset("unum_open failed")) { return; }
     UnicodeString uExpected = UnicodeString::fromUTF8(utf8Expected);
-    double expected = unum_parseDouble(nf, uExpected.getBuffer(), uExpected.length(), 0, pErrorCode);
+    double expected = unum_parseDouble(nf, uExpected.getBuffer(), uExpected.length(), 0, status);
     unum_close(nf);
+    if (status.errIfFailureAndReset("unum_parseDouble(\"%.*s\") failed", uExpected.length(),
+                                    uExpected.getBuffer())) {
```

Comment:
These edits were mine back in March, I see I made a mistake: format strings cannot handle UnicodeStrings (at least not with %s).

Can you change it to use utf8Expected instead, or should I push a commit to do so? Like so:

        if (status.errIfFailureAndReset("unum_parseDouble(\"%s\") failed",
                                    utf8Expected)) {
          return;
        }

Reply:
Done!